student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name)
+    super(age, name)
+    @color = classroom
```

Comment:
[OPTIONAL]
- [ ] To be in sync with variable names and not get confused when the app gets bigger, kindly rename your `@color` instance to `@classroom`. It should look something like this `@classroom = classroom`

Reply:
Done!